web/src/popovers.js

```diff
@@ -1105,11 +1112,12 @@ export function hide_all_except_sidebars(opts) {
 
 // This function will hide all the popovers, including the mobile web
 // or narrow window sidebars.
-export function hide_all(not_hide_tippy_instances) {
+export function hide_all(not_hide_tippy_instances, dont_hide_tippy_dropdowns = false) {
```

Comment:
i see `not_hide_tippy_instances` is only used once.
we can convert this to `hide_all(opts)` similar to `hide_all_except_sidebars(opts)`
might as well rename the 2 variables to be similar.